src/window/editor/empire.c

```diff
@@ -310,6 +350,8 @@ static void handle_input(const mouse *m, const hotkeys *h)
     }
     if (h->toggle_editor_battle_info) {
         data.show_battle_objects = !data.show_battle_objects;
+    } else if (h->f5_pressed) {
```

Comment:
For this to work properly,  you'll need to disable the default `F5` behaviour of centering the window.